bots/run-queue

```diff
@@ -26,11 +26,64 @@ import ssl
 import subprocess
 import sys
 import time
+import logging
 
 import pika
 from task import redhat_network
 
+# Returns a command to execute
```

Comment:
It actually returns a pair (cmd_argv,  body).

Reply:
woops you're right, didn't update the comment.